.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<!-- Thanks for submitting a PR! Please ensure the following requirements are met when writing a new post -->
+
+## New Post Checklist
+
+- [ ] Links are linkified
+  - [ ] Bare links are surrounded by `<>` (required since Zola follows commonmark strictly)
```

Comment:
More like because `pulldown_cmark` doesn't support auto-detection of bare links: https://github.com/raphlinus/pulldown-cmark/issues/695

Reply:
Oh! Well that's unfortunate. Reading the spec I thought this is actually intentional by the commonmark spec 😅